components/hab/src/command/studio.rs

```diff
@@ -214,6 +215,12 @@ mod inner {
             .into());
         cmd_args.push("--volume".into());
         cmd_args.push(format!("{}:/src", env::current_dir().unwrap().to_string_lossy()).into());
+
+        if let Ok(ops) = henv::var(DOCKER_OPS) {
+            let ops = ops.split(" ").map(|v| v.into()).collect::<Vec<_>>();
```

Comment:
I have some concerns that this may be a bit ham-handed to split on spaces, but if the whitespace isn't purged then extraneous docker errors are encountered such as:

```
["run", "--rm", "--tty", "--interactive", "--privileged", "--env", "HAB_ORIGIN=hoverbear", "--volume", "/var/run/docker.sock:/var/run/docker.sock", "--volume", "/Users/hoverbear/.hab/cache/keys:/hab/cache/keys", "--volume", "/Users/hoverbear/git/habitat/components/hab:/src", "-p 8080:8080", "habitat-docker-registry.bintray.io/studio:0.11.0", "enter"]
/usr/local/bin/docker: Invalid hostPort:  8080.
```